verilog/formatting/tree_unwrapper_test.cc

```diff
@@ -2637,6 +2637,81 @@ const TreeUnwrapperTestData kUnwrapPreprocessorTestCases[] = {
         L(0, {"`endif"}),
     },
 
+    {
```

Comment:
In these new tests, comment that they depend on expansion of property_spec's inside the macro.  (We know immediately why these tests are more advanced than those using plain expressions.)

Reply:
Fixed